xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-filters/xwiki-platform-notifications-filters-default/src/main/java/org/xwiki/notifications/filters/migration/R160300000XWIKI17243DataMigration.java

```diff
@@ -120,33 +126,57 @@ public String getDescription()
     @Override
     public XWikiDBVersion getVersion()
     {
-        return new XWikiDBVersion(160000000);
+        return new XWikiDBVersion(160300000);
     }
 
     @Override
     protected void hibernateMigrate() throws DataMigrationException, XWikiException
     {
         String statement = "from doc.object(XWiki.WatchListClass) as watchListDoc";
-        int offset = 0;
-        List<String> results;
-        do {
-            try {
-                results = this.queryManager.createQuery(statement, Query.XWQL)
-                    .addFilter(uniqueFilter)
-                    .setOffset(offset)
-                    .setLimit(BATCH_SIZE).execute();
-
-                this.logger.info("Found [{}] users with WatchListClass objects... migrating them.", results.size());
-                for (String result : results) {
-                    DocumentReference watchListDoc = this.documentReferenceResolver.resolve(result);
-                    this.migrateWatchListDoc(watchListDoc);
+        Long numberOfEntries = -1L;
```

Comment:
It's not clear to be why this is a `Long` and not a `long`. It does not seem to ever be set to null.

Reply:
No particular reason, I just wrote that after the `List<Long>`. Switching to `long`